packages/orchestration/src/orchestration-api.ts

```diff
@@ -150,10 +148,12 @@ export interface Chain<CI extends Partial<ChainInfo>> {
     : never;
   // FUTURE supply optional port object; also fetch port object
 
-  query: CI extends { icqEnabled: true }
+  query: CI extends { chainId: string; icqEnabled: true }
```

Comment:
Oops, artifact. Only the one on L153 is needed. Without L153, we see: "Type '"chainId"' cannot be used to index type 'CI'.ts(2536)"